api/fixtures/example.go

```diff
@@ -408,6 +409,10 @@ func (o ExampleOptions) Resources() *ExampleResources {
 		},
 	}
 
+	if o.BaseDomainPrefix != "" {
```

Comment:
how can I set an empty basedomainprefix?

Reply:
using the cli, its not possible to distinguish whether the flag is not set or set to an explicitly empty string. I thought we could check if the prefix is set to a special value e.g. `none`, and translate that to an empty string pointer in the API.

I wasn't sure if we really want to support this niche use-case in the cli anyway. Do you think I should implement that?

Reply:
It'd be good so we can test it, given that service delivery is likely to use an empty prefix.